scripts/packaging/package.snapshot

```diff
@@ -166,24 +199,30 @@ Source/ThirdParty/IOS/Sentry.framework/PrivateHeaders/SentryLog.h
 Source/ThirdParty/IOS/Sentry.framework/PrivateHeaders/SentryRequestOperation.h
 Source/ThirdParty/IOS/Sentry.framework/Sentry
 Source/ThirdParty/Linux/
+Source/ThirdParty/Linux/bin/
 Source/ThirdParty/Linux/bin/crashpad_handler
 Source/ThirdParty/Linux/bin/libsentry.dbg.so
 Source/ThirdParty/Linux/bin/libsentry.so
+Source/ThirdParty/Linux/include/
 Source/ThirdParty/Linux/include/sentry.h
 Source/ThirdParty/Mac/
+Source/ThirdParty/Mac/bin/
 Source/ThirdParty/Mac/bin/crashpad_handler
 Source/ThirdParty/Mac/bin/libsentry.dylib
 Source/ThirdParty/Mac/bin/libsentry.dylib.dSYM/
+Source/ThirdParty/Mac/bin/libsentry.dylib.dSYM/Contents/
 Source/ThirdParty/Mac/bin/libsentry.dylib.dSYM/Contents/Info.plist
 Source/ThirdParty/Mac/bin/libsentry.dylib.dSYM/Contents/Resources/
+Source/ThirdParty/Mac/bin/libsentry.dylib.dSYM/Contents/Resources/DWARF/
 Source/ThirdParty/Mac/bin/libsentry.dylib.dSYM/Contents/Resources/DWARF/libsentry.dylib
+Source/ThirdParty/Mac/include/
 Source/ThirdParty/Mac/include/sentry.h
 Source/ThirdParty/Win64/
+Source/ThirdParty/Win64/bin/
 Source/ThirdParty/Win64/bin/crashpad_handler.exe
 Source/ThirdParty/Win64/bin/sentry.dll
 Source/ThirdParty/Win64/bin/sentry.pdb
+Source/ThirdParty/Win64/include/
 Source/ThirdParty/Win64/include/sentry.h
+Source/ThirdParty/Win64/lib/
 Source/ThirdParty/Win64/lib/sentry.lib
-Source/ThirdParty/CLI/sentry-cli-Darwin-universal
-Source/ThirdParty/CLI/sentry-cli-Linux-x86_64
-Source/ThirdParty/CLI/sentry-cli-Windows-x86_64.exe
```

Comment:
FYI @tustanivsky - when you need to make changes to this file, run the following locally. 
```shell-script
pwsh ./scripts/packaging/pack.ps1
pwsh ./scripts/packaging/test-contents.ps1 accept
```
(it sorts the output correctly so in this case the manual changes got sorted in this PR even though they don't belong here)